[IAM] Update manage.md (#11846)
cloudflare/cloudflare-docs: content/fundamentals/setup/manage-members/manage.md

[^1]: {{<render file="_account-member-manage-limitation.md">}}

> {{<Aside type="note">}}
> If you have been invited to an account, you can remove yourself from this account under **Manage Account > Members**.
> You can thee select click on your email address and then click **Leave**.
> {{</Aside>}}
> 

Reviewer: ```suggestion
{{<Aside type="note">}}
If you have been invited to an account and want to remove yourself from the account, go to **Manage Account > Members**. Under your email address, select **Leave**.
{{</Aside>}}

```
